### ISSUE TITLE
Resolution Scaling not working on FEW 3 Hopes

### ISSUE THREAD

**Issue by BobJink22** [Ryujinx/Ryujinx#3465]
## Bug Report

Resolution of the games revert to the switch native one and gets stuck on it after loading the camp area.
Once the resolution is "reverted" to the native resolution of the switch (1080p in docked mode), it can't be changed to anything else, even when manually editing the settings again.
the resolution scaling chosen in the settings (x2, x3, x4) is however still displayed next to the "Handheld/Docked" button.

### How can the issue be reproduced?
Start the game and load a save, with resolution set to x2 in the emulator settings, the game will be at the expected 4K resolution (Docked)
- if the game immediatelly load the camp area, the resolution is not changed yet, but will be the next time the area is loaded (going into mock battle then returning to camp will lower the resolution back to native uppon loading the camp)
- joining a battle doesn't seem to change the resolution, only 
- if the game load on a cutscene, the cutscene plays in the expected resolution, once it ends and the game load the camp, the resolution is changed back to the native resolution of the Switch

### Log file
[Ryujinx_1.1.178_2022-07-13_22-10-53.log](https://github.com/Ryujinx/Ryujinx/files/9105898/Ryujinx_1.1.178_2022-07-13_22-10-53.log)

### Environment?
- Ryujinx version: 1.1.178 (also happened on earlier versions)
- Game version: 1.0.0
- System Specs:
  - OS: Windows 10
  - CPU: R7 3700x
  - GPU: NVIDIA RTX 2080ti
  - RAM: 16GB
- Applied Mods : None

### Additional context?
No apparent issue on other games after a quick trial (Mario Odyssey)

Exhibit 1, the game load immediatelly on the camp, the resolution is 4K as expected (native x2, docked)
![ryujinx_capture_2022-07-13_22-28-15](https://user-images.githubusercontent.com/109244509/178830885-f8d1c4ae-0b64-4e56-8f64-d2f88f0f1993.png)
Exhibit 2, after joining a battle and then returning to camp, the resolution is now 1080p
![ryujinx_capture_2022-07-13_22-30-33](https://user-images.githubusercontent.com/109244509/178831147-07b50fd7-e841-4282-b994-2dee8ee79fd7.png)
(differences are mostly noticable on characters outline)

**Comment by Niwu34** [comment]
Maybe try this Pr with it:
https://github.com/Ryujinx/Ryujinx/pull/2860

**Comment by BobJink22** [comment]
Yep, https://github.com/Ryujinx/Ryujinx/pull/2860 seems to do the trick.
I could join a mock battle, then back to the camp, and the resolution held, I can also change it in real time which I couldn't before.

**Comment by MetrosexualGarbodor** [comment]
That PR is not going to get merged, and I'm not sure if it fixes the issue because it actually fixes it or because it's just a build from before 1.1.149, which is where resolution scaling started resetting for this game.

**Comment by SnowMaw** [comment]
Vulkan API fixed this problem

**Comment by MetrosexualGarbodor** [comment]
Please post about this in #1488 if it still occurs.
